doc-Managing_Providers/topics/Automation_Management_Providers.adoc

```diff
@@ -1,30 +1,42 @@
 [[automation_management_providers]]
 = Automation Management Providers
 
+[NOTE]
+====
+{product-title} ships with the Ansible role disabled. Navigate to menu:settings menu[Configuration > Settings] and select the desired server under *Zones*. Set the *Server Role* for *Embedded Ansible* to *On* to enable the role.
```

Comment:
@cbudz Can you rephrase the first line to something like "In {product-title}, the Ansible role is disabled by default." What do you think? We use "include(s|d)" instead of "ship" as in the style guide, though that also can be avoided here I think.